modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImport/ModalImportWarning.tsx

```diff
@@ -49,6 +52,31 @@ export function ModalImportWarning({
 						}
 					)}
 
+					{Liferay.FeatureFlags['LPS-187142'] &&
+						alreadyImportedObjectDefinitions &&
+						alreadyImportedObjectDefinitions.map(
+							(objectDefinition) => (
+								<li key={objectDefinition.name}>
+									{stringUtils.getLocalizableLabel(
+										objectDefinition.defaultLanguageId,
+										objectDefinition.label,
+										objectDefinition.name
+									)}
+								</li>
+							)
+						)}
+
+					<br />
```

Comment:
Hey @danvb112, what do you think in insert this `< br/>`  inside the FF escope. Do avoid change the current style of `WarningModal`.

Reply:
Hey @igor-franca, sure! I will unify this FF logic in only one code validation